examples/migrating_trench/trench_example.py

```diff
@@ -0,0 +1,168 @@
+"""
+Migrating Trench Test case
+=======================
+
+Solves the test case of a migrating trench.
+
+We use this test case to validate the implementation of the mathematical and
+numerical methods used in Thetis to model sediment transport and morphological changes.
+In the figure produced, we compare our results with experimental data from a lab study
+
+For more details, see
+[1] Clare et al. 2020. “Hydro-morphodynamics 2D Modelling Using a Discontinuous
+    Galerkin Discretisation.” EarthArXiv. January 9. doi:10.31223/osf.io/tpqvy.
+
+"""
+
+from thetis import *
+
+import numpy as np
+import pandas as pd
```

Comment:
pandas is only used to read in csv files, this can be done with numpy as well. I'd remove the dependency (nothing in thetis depends on pandas).

Reply:
rm pandas